tests/test_asyncio/test_commands.py

```diff
@@ -2530,33 +2556,22 @@ async def test_georadius_with(self, r: valkey.Valkey):
 
         # test a bunch of combinations to test the parse response
         # function.
-        assert await r.georadius(
+        georadius_result = await r.georadius(
             "barcelona",
             2.191,
             41.433,
             1,
             unit="km",
-            withdist=True,
-            withcoord=True,
-            withhash=True,
-        ) == [
-            [
-                b"place1",
-                0.0881,
-                3471609698139488,
-                (2.19093829393386841, 41.43379028184083523),
-            ]
-        ]
-
-        assert await r.georadius(
-            "barcelona", 2.191, 41.433, 1, unit="km", withdist=True, withcoord=True
-        ) == [[b"place1", 0.0881, (2.19093829393386841, 41.43379028184083523)]]
+            **georadius_kwargs,
+        )
 
-        assert await r.georadius(
-            "barcelona", 2.191, 41.433, 1, unit="km", withhash=True, withcoord=True
-        ) == [
-            [b"place1", 3471609698139488, (2.19093829393386841, 41.43379028184083523)]
-        ]
+        if "withcoord" in georadius_kwargs:
```

Comment:
I'd also add a check on `georadius_result` lenght e.g.
```suggestion
        assert len(georadius_result) == 1
        if "withcoord" in georadius_kwargs:
```